README.md

```diff
@@ -11,6 +11,23 @@ The Node.js Foundation’s new [Medium](https://medium.com/) publication serves
 - Reports on Node.js and JavaScript events (these events must have a code of conduct to be considered)
 - Other topics will certainly be considered, but it should be something of interest to the Node.js community
 
+## Editors
+We have list of people who review the post before publishing. If the post is specific to some technical topic, we seek review from someone who has knowledge of that domain.
+
+### How to become an editor?
+We are looking for more reviewers. If you are interested, please open an issue in [nodejs-collection repo](https://github.com/nodejs/nodejs-collection/issues) with your details and the team will review it.
```

Comment:
What information needs to be included to be an editor?

Reply:
I think it should include,
- Intro about themself.
- How much time they can give.
- Area of expertise.
- Email address for contact.

What do you suggest?